### ISSUE TITLE
Remember expanded nodes of tree?

### ISSUE THREAD

**Issue by kkovac1** [josdejong/jsoneditor#1253]
Hello, is it possible to remember expanded nodes somehow in session/local storage. So the goal is when I refresh the page I want to remember expanded nodes.

**Comment by josdejong** [comment]
Yes that would be great! With the current editor this is not possible: the history also contains DOM nodes which cannot be serialized.

I'm working on a completely new JSONEditor though, where this will become possible: the state contains just JSON which can be serialized. See: https://github.com/josdejong/svelte-jsoneditor
